typescript/packages/orb/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@subconsciousnetwork/orb",
+  "author": "Subconscious Inc.",
+  "license": "Apache-2.0 OR MIT",
+  "version": "0.1.0",
+  "description": "A batteries-included JavaScript package for accessing the Noosphere",
+  "type": "module",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "wireit",
+    "generate_wasm": "wireit",
+    "clean": "wireit",
+    "test": "wireit"
+  },
+  "dependencies": {
+    "@web/dev-server": "^0.1.34",
+    "esbuild": "^0.15.9",
+    "typescript": "~4.8.2",
+    "wireit": "^0.8.0"
+  },
+  "wireit": {
+    "build": {
```

Comment:
See https://github.com/google/wireit#typescript for a bunch of suggestions for `tsc` that can improve performance.